operator/scripts/scaling/scalingAction.sh

```diff
@@ -144,14 +144,31 @@ get_custom_resource_domain() {
     -v \
     --cacert /var/run/secrets/kubernetes.io/serviceaccount/ca.crt \
     -H "Authorization: Bearer $(cat /var/run/secrets/kubernetes.io/serviceaccount/token)" \
-    $kubernetes_master/apis/weblogic.oracle/$domain_api_version/namespaces/$wls_domain_namespace/domains/$wls_domain_uid)
+    "$kubernetes_master"/apis/weblogic.oracle/"$domain_api_version"/namespaces/"$wls_domain_namespace"/domains/"$wls_domain_uid")
   if [ $? -ne 0 ]; then
     trace "Failed to retrieve WebLogic Domain Custom Resource Definition"
     exit 1
   fi
   echo "$DOMAIN"
 }
 
+# Retrieve Cluster Custom Resource
+# args:
+# $1 Cluster name (assumed to be same as name of the Cluster Resource)
```

Comment:
Why can we assume the CR name is the same as the cluster name?

I assume the CR name would normally be decorated with the domain-uid, where-as the cluster name would not...

Reply:
@lennyphan can you help confirm if the Cluster resource name is <domain-uid>-<cluster-name>? Thanks

Reply:
Not sure. @rjeberhard ?

Reply:
Talked with Lenny. He suggested we have to search for a matching Cluster CR with matching domainUID and clusterName.

Reply:
Makes sense to me.

Reply:
The `metadata.name` of the Cluster resource can be anything and doesn't have to match the cluster name. In the work I'm doing in the webhook to create Cluster resources as we upgrade v8 domains, the webhook will create Cluster resources that have a <domain-uid>-<cluster-name> decorated name as described.

However, since that's not required. The only foolproof way to find a Cluster resource for a given cluster is to see what Cluster resources are referenced from the Domain and then see which of these has the required cluster name.

Reply:
Updated script to do what Ryan and Lenny suggested